com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -1997,10 +1997,10 @@ void GetEnvLightVolumeDataAndBound(HDProbe probe, LightVolumeType lightVolumeTyp
             m_lightList.lightsPerView[viewIndex].lightVolumes.Add(lightVolumeData);
         }
 
-        void AddBoxVolumeDataAndBound(OrientedBBox obb, LightCategory category, LightFeatureFlags featureFlags, Matrix4x4 worldToView, int viewIndex, bool isProbeVolume = false, float normalBiasDilation = 0.0f)
+        void CreateBoxVolumeDataAndBound(out LightVolumeData volumeData, out SFiniteLightBound bound, OrientedBBox obb, LightCategory category, LightFeatureFlags featureFlags, Matrix4x4 worldToView, float normalBiasDilation = 0.0f)
```

Comment:
Note: in our convention we put "out" parameter at the end of the list, not the beginning

Reply:
I can update it - how does that work with optional parameters at the end? normalBiasDilation defaults to 0.0 if not specified right now. I can just get rid of the optional parameter and make the call site explicitly specify 0.0 for the non-probe volume case if necessary. Mostly curious.